scripts/test.sh

```diff
@@ -410,15 +410,10 @@ function lint_fix_pass {
   run_for_all_workspace_modules golangci-lint run --config "${ETCD_ROOT_DIR}/tools/.golangci.yaml" --fix
 }
 
-function license_header_per_module {
-  # bash 3.x compatible replacement of: mapfile -t gofiles < <(go_srcs_in_module)
-  local gofiles=()
-  while IFS= read -r line; do gofiles+=("$line"); done < <(go_srcs_in_module)
-  run_go_tool "github.com/google/addlicense" --check "${gofiles[@]}"
-}
-
 function license_header_pass {
```

Comment:
Could we simplify further and implement https://golangci-lint.run/docs/linters/configuration/#goheader via golangci-lint and simply scan the entire repository rather than do things per module with complex shell scripts?

Reply:
Great idea!